src/context/chat-context.tsx

```diff
@@ -0,0 +1,35 @@
+import { createContext, useContext, useState, useMemo } from 'react'
+import { Offer } from '~/types'
+
+interface ChatProviderProps {
+  children: React.ReactNode
```

Comment:
do you really need to create interface for this, you can set ReactNode directly to children

Reply:
Fixed